_posts/0000-01-02-Bincell.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
:smile:~~this~~http://www.github.com/ @Bincell 
#1
mojombo/github-flavored-markdown#1

mojombo/github-flavored-markdown@16c999e8c71134401a78d4d46435517b2271d6ac
First Header | Second Header
------------ | -------------
Content from cell 1 | Content from cell 2
Content in the first column | Content in the second column
- [x] @mentions, #refs, [links](), **formatting**, and <del>tags</del> supported
- [x] list syntax required (any unordered or ordered list supported)
- [x] this is a complete item
- [ ] this is an incomplete item
*This text will be italic*